docs/EVM/precompiles/dapp-staking.md

```diff
@@ -0,0 +1,39 @@
+### DappsStaking Precompile
+
+DappsStaking Precompile enables EVM smart contract to access `pallet-dapps-staking` functionality.
+Example use of this precompile from Contract A:
+
+```
+import "./DappsStaking.sol";
+contract A {
+    DappsStaking public constant DAPPS_STAKING = DappsStaking(0x0000000000000000000000000000000000005001);
+
+    /// @notice Check current era
+    function checkCurrentEra() public view {
+        uint256 currentEra = DAPPS_STAKING.read_current_era();
+    }
+}
+```
+
+Example use: check `current era` and `total staked amount` in the `pallet-dapps-staking` for Shiden Network. For this example we will use Remix.
+
+1. Copy `DappsStaking.sol` from [astar-frame repo](https://github.com/AstarNetwork/astar-frame/) and create new contract in Remix:
+![](https://i.imgur.com/mr0TcLq.png)
+
+2. Compile the dAppStaking contract:
+![](https://i.imgur.com/6Wgg9rf.jpg)
+
+3. The precompile does not need to be deployed since it is already on the network, but you need to tell Remix where to find it.
```

Comment:
Perhaps this information should be under a dedicated chapter or page?

There's no guarantee that someone starting with precompiles will first check dapps-staking. Having an explanation on how to test this via Remix is useful on its own, imo.

Reply:
Good point. Originally that text was on precompile page after the tables, but I moved it to a dedicated page.